webapp/src/pages/ChatroomPage/ChatroomPage.tsx

```diff
@@ -37,10 +31,7 @@ type ChatroomProps = {
 
 function Chatroom({ chatroomId, authUser }: ChatroomProps) {
   const { goBack } = useNavigation();
-  const [status, setStatus] = useState<SubmitStatus>({
-    type: 'pending',
-    message: '',
-  });
+  const { warn } = useNotificationContext();
```

Comment:
praise: Thank you! Looks great!

suggestion: There are some CSS rules we don't use anymore:

<details>

```diff
diff --git a/webapp/src/pages/ChatroomPage/ChatroomPage.css b/webapp/src/pages/ChatroomPage/ChatroomPage.css
index 1184e70..d8bac03 100644
--- a/webapp/src/pages/ChatroomPage/ChatroomPage.css
+++ b/webapp/src/pages/ChatroomPage/ChatroomPage.css
@@ -4,20 +4,15 @@
   background-color: var(--color-background);
   color: var(--color-light);
   display: grid;
+  gap: 1rem;
   grid-template-areas:
     'header'
     'messages'
-    'message-input'
-    'status';
-  grid-template-rows: auto 1fr auto auto;
+    'message-input';
+  grid-template-rows: auto 1fr auto;
   grid-template-columns: 1fr;
 }
 
-.chatroom-header,
-.chatroom-messages {
-  margin-bottom: 1rem;
-}
-
 .chatroom-loading {
   grid-column: 1/-1;
   grid-row: 1/-1;
diff --git a/webapp/src/pages/LoginPage/LoginPage.css b/webapp/src/pages/LoginPage/LoginPage.css
index e5637fe..bc278a0 100644
--- a/webapp/src/pages/LoginPage/LoginPage.css
+++ b/webapp/src/pages/LoginPage/LoginPage.css
@@ -60,11 +60,6 @@
   width: 100%;
 }
 
-.login-form > p:empty::before {
-  content: '';
-  display: inline-block;
-}
-
 .final-text {
   display: flex;
   justify-content: center;
diff --git a/webapp/src/pages/RegisterPage/RegisterPage.css b/webapp/src/pages/RegisterPage/RegisterPage.css
index 8eacdf1..091d06f 100644
--- a/webapp/src/pages/RegisterPage/RegisterPage.css
+++ b/webapp/src/pages/RegisterPage/RegisterPage.css
@@ -47,11 +47,6 @@
   max-width: 20rem;
 }
 
-.register-form .inputs-container > p:empty::before {
-  content: '';
-  display: inline-block;
-}
-
 .register .landing-animation {
   display: none;
 }

```

</details>

Reply:
done! thanks for the suggestions